_includes/tutorials/time-concepts/ksql/markup/test/make-test-output.adoc

```diff
@@ -0,0 +1,5 @@
+Next, create a file at `test/output.json` with the expected outputs:
```

Comment:
We don't use this either. Can we remove it?

Reply:
Removed